docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,147 @@
+commenting out all the default NTP sources (lines that start with `pool` or 
+`server`). 
+Then, add the following lines to the configuration file to use the Chrony 
+charm as the time source. 
+Make sure to change the Chrony charm address in the example (10.81.171.230) 
+to the actual address used in your deployment. 
+You can find the correct address using the `juju status` command.
+
+```
+server 10.81.171.230 iburst
+```
+
+After modifying the Chrony configuration file, run `sudo systemctl restart chrony`
+to apply changes.
+
+## Enable NTS
+The NTS protocol is a security enhancement over the NTP protocol. 
+It adds authentication to the NTP protocol to prevent time results from being 
+tampered with by a malicious actor. 
+To enable the NTS protocol in the Chrony charm, provide a TLS certificate to 
+the Chrony charm.
+
+You can either use the `ntp-certificates` configuration of the Chrony charm 
+or a `tls-certificate` provider charm, and providing the certificate via 
+Juju integration.
+In this example, we will use the `self-signed-certificates` charm to provide 
+the TLS certificate. 
+
+Configure the Chrony charm's `server-name` configuration with a test value 
+(ntp.example.com), as it is required for the `tls-certificate` integration, 
+then deploy the `self-signed-certificates` charm and relate it to the Chrony 
+charm.
+
+```
+juju config chrony server-name=ntp.example.com
+juju deploy self-signed-certificates
+juju integrate chrony self-signed-certificates
+```
+
+Next, we will retrieve the root certificate from the `self-signed-certificates`
+charm and use it to configure the Chrony client to use NTS when communicating 
+with the Chrony charm.
+
+
+```
+juju run self-signed-certificates/0 get-ca-certificate | \
+  awk '{gsub(/^ */, "")} /-----BEGIN CERTIFICATE-----/{print; flag=1; next} /-----END CERTIFICATE-----/{print; flag=0} flag {print}' | \
+  sudo tee /etc/chrony/nts.crt && sudo chown _chrony:_chrony /etc/chrony/nts.crt
+```
+
+You will need to add an entry to `/etc/hosts` with the `server-name` value we 
+just set and the IP address of the Chrony charm.
+
+```
+10.81.171.230 ntp.example.com
+```
+
+After that, enable NTS in the Chrony client configuration and restart the 
+Chrony client.
+
+```
+server ntp.example.com nts iburst certset 1
+ntstrustedcerts 1 /etc/chrony/nts.crt
+```
+
+Wait a few seconds, and you should see that the Chrony client is synchronized 
+with the Chrony charm.
+
+```
+chronyc tracking
+Reference ID    : 0A51AB43 (ntp.example.com)
+Stratum         : 4
+Ref time (UTC)  : Fri Sep 13 15:47:07 2024
+System time     : 0.000000026 seconds slow of NTP time
+Last offset     : +0.000024886 seconds
+RMS offset      : 0.000024886 seconds
+Frequency       : 31.247 ppm slow
+Residual freq   : +3.838 ppm
+Skew            : 67.618 ppm
+Root delay      : 0.231240883 seconds
+Root dispersion : 0.015818920 seconds
+Update interval : 2.0 seconds
+Leap status     : Normal
+```
```

Comment:
Add a "Clean up the environment" section to provide a satisfying end to the tutorial :)

Reply:
Yes, I think this is a good idea. However, I’m not sure how much cleanup we can accomplish for this tutorial. The issue here  that the `chrony` package we installed via `apt` is somewhat "destructive". Installing it will replace the system’s default NTP client with `chrony`, and I’m unsure if uninstalling it will restore the original NTP client. Cleanup could involve reverting the changes made to the `chrony` configuration file on the host system back to its default settings, while leaving `chrony` installed. This should allow the reader’s system to functionally back to before, especially since on ubuntu 25.04 and above, `chrony` becomes the default NTP client anyway, so no changes will occur in that case. Alternatively, we could consider uninstalling `chrony` and fully restoring the system to its prior state, which is more complex. To simplify this, we might revise the tutorial to install the `chrony` NTP client within an LXD VM rather than directly on the host machine.

Reply:
Thank you for your thoughts and the explanation why a cleanup section is tricky to include. I'm leaning toward installing the `chrony` NTP client within a VM, but my question to you is, how much extra work would it be for you to rework the tutorial?

Reply:
I have updated the document to install chrony as the NTP client in an LXD virtual machine. To differentiate which commands should run on the host machine and which should run on the virtual machine, I used the command line prompts to clarify this (`ubuntu@laptop:~$` vs `root@ntp-client:`). I have also added a section at the end for cleanup, which is very simple: just delete the virtual machine and the Juju model.

Reply:
Thanks so much for updating! Typically we're not supposed to use command line prompts (according to the style guide), but I think it's fine in this case since we need to distinguish between two different machines.